src/app/mail/mail-sidebar/compose-mail-dialog/compose-mail-dialog.component.ts

```diff
@@ -29,6 +29,7 @@ export class ComposeMailDialogComponent implements AfterViewInit {
   isMinimized: boolean;
   isFullScreen: boolean;
   private confirmModalRef: NgbModalRef;
+  mailSubject = 'New message';
```

Comment:
why this default value?

Reply:
It was not needed, but I was thinking when there is not subject added the default value will be showed, but I had already added a condition on html, So I have updated it.